### PR TITLE
Cp integration 2.8.0rc6

### PR DESCRIPTION
**Pull request by kcelia**
Recently, the sympy library was upgraded, which caused mypy type-checking to fail with the following errors:

```
find ./tests/ -name "*.py" | xargs poetry run mypy --ignore-missing-imports --implicit-optional --check-untyped-defs
tests/quantization/test_quantized_ops.py:379: error: Argument "q_input" to "quantized_op_results_are_equal" has incompatible type "Tuple[Any, Any]"; expected "Optional[ndarray[Any, Any]]"  [arg-type]
tests/quantization/test_quantized_ops.py:592: error: Argument "q_input" to "quantized_op_results_are_equal" has incompatible type "Tuple[Any, Any]"; expected "Optional[ndarray[Any, Any]]"  [arg-type]
Found 2 errors in 1 file (checked 36 source files)
``` 

Updating the signature of  `def quantized_op_results_are_equal` in `tests/quantization/test_quantized_ops.py` file after sympy upgrade.

Steps:
- Edit pypproject.toml with the new cp version
- poetry lock --no-update
- make sync_env
- push

Check potential package upgrade

### CODE REVIEW

File: docs/guides/using_gpu.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ a model is compiled for CUDA, executing it on a non-CUDA-enabled machine will ra
 
 | Feature     | Built-in models | Custom Models | Deployment | DataFrame |
 | ----------- | --------------- | ------------- | ---------- | --------- |
-| GPU support | ✅               | ✅             | ✅          | ❌         |
+| GPU support | ✅              | ✅            | ✅         | ❌        |
```
Comment (andrei-stoian-zama) [inline docs/guides/using_gpu.md:12]: this change causes mdformat checker to fail

```
make[1]: *** [Makefile:628: check_mdformat] Error 123
```

**Review by andrei-stoian-zama — CHANGES_REQUESTED** [review]

File: docs/guides/using_gpu.md (hunk 1 above)
Comment (kcelia) [reply docs/guides/using_gpu.md]: I reinstalled my venv, but I still get this change

Comment (kcelia) [reply docs/guides/using_gpu.md]: I found the problem, it was caused by a bad rebasing on main.

File: deps_licenses/licenses_linux_user.txt [hunk 2]
```diff
@@ -32,35 +32,35 @@ nvidia-curand-cu12, 10.3.2.106, Other/Proprietary License
...
 psutil, 5.9.8, BSD License
 python-dateutil, 2.9.0.post0, Apache Software License; BSD License
-pytz, 2024.1, MIT License
+pytz, 2024.2, MIT License
 requests, 2.32.3, Apache Software License
 scikit-learn, 1.1.3, BSD License
 scipy, 1.10.1, BSD License
 six, 1.16.0, MIT License
-skl2onnx, 1.12, Apache Software License
+skl2onnx, 1.17.0, Apache Software License
```
Comment (bcm-at-zama) [inline deps_licenses/licenses_linux_user.txt:52]: did you poetry lock run with the no-update? I am surprised there are so many dep changes

Comment (kcelia) [reply deps_licenses/licenses_linux_user.txt]: thanks for the reminder, i forgot the --no update

**Comment by kcelia** [comment]
That doesn't smell good

**Comment by andrei-stoian-zama** [comment]
Not needed anymore

**Pull request closed without merging**